<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>755</width>
    <height>587</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 255, 29);
background-color: rgb(11, 11, 11);
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTabWidget" name="tab_all">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_fxShot">
     <attribute name="title">
      <string>Fx</string>
     </attribute>
     <widget class="QGroupBox" name="user_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>271</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Elements</string>
      </property>
      <widget class="QListWidget" name="listElement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>251</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineElement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>type new element and press enter..</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>271</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Shot list</string>
      </property>
      <widget class="QListWidget" name="shotList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>301</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>301</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Sequence</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="seqBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Select sequence.</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="user">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>271</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
      </property>
      <property name="title">
       <string>User</string>
      </property>
      <widget class="QListWidget" name="listUser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>131</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_fxAsset">
     <attribute name="title">
      <string>Asset</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>Fx Elements</string>
      </property>
      <widget class="QLineEdit" name="lineElem_asset">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>type new fx element and press enter..</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_asset">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>271</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>231</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>User</string>
      </property>
      <widget class="QListWidget" name="listUser_asset">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>211</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_light">
     <attribute name="title">
      <string>Lighting</string>
     </attribute>
     <widget class="QGroupBox" name="user_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Elements</string>
      </property>
      <widget class="QListWidget" name="lgt_listElement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>271</width>
         <height>151</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lgt_lineElement">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>type new element and press enter..</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="user_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>User</string>
      </property>
      <widget class="QListWidget" name="lgt_listUser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>271</width>
         <height>131</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Sequence</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QComboBox" name="lgt_seqBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Select sequence.</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>271</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Shot list</string>
      </property>
      <widget class="QListWidget" name="lgt_shotList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>301</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>301</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.340909 rgba(38, 46, 43, 165), stop:0.988636 rgba(32, 123, 255, 255), stop:1 rgba(102, 255, 126, 255));</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>551</width>
      <height>81</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Project</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Show  :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="job">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Job     :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="showBox">
        <property name="cursor">
         <cursorShape>ClosedHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(231, 46, 0);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="jobPath">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(1, 1, 1, 0);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Houdini project path</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>91</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.340909 rgba(38, 46, 43, 165), stop:0.988636 rgba(32, 255, 95, 255));</string>
   </property>
   <property name="text">
    <string>SET SHOT</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imageX">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>630</y>
     <width>431</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="prjpath">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Project Path</string>
   </property>
   <widget class="QRadioButton" name="prjpath1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>filmServe</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="prjpath2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TVC_Beijing</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listElement</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listUser_asset</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lgt_listElement</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>pushButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
